src/tools/compiletest/src/main.rs

```diff
@@ -1,5 +1,6 @@
 #![crate_name = "compiletest"]
 #![feature(test)]
+#![feature(path_buf_capacity)]
```

Comment:
Could this use existing stable methods instead of adding a new unstable feature?

Reply:
Done!  Can't reserve capacity without it, but that's an unimportant optimization at best.